FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -248,15 +233,19 @@ const GeneralInfo = (props) => {
                             value={user.edrpou}
                             error={edrpouError}
                         />
-                        <MultipleSelectChip
+                        {!isLoading && 
+                        <OneSelectChip
                             name='regions'
-                            options={REGIONS}
+                            options={fetchedRegions}
                             label={LABELS.regions}
-                            updateHandler={onUpdateSelectField}
+                            updateHandler={onUpdateOneSelectField}
                             requredField={false}
-                            value={user.regions}
                             defaultValue="Оберіть"
+                            value={fetchedRegions.find((el) => el.key ===  user.regions) 
```

Comment:
It's better to use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) and [optinal chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) `fetchedRegions.find((el) => el.key ===  user.regions)?.value ?? ''`.

Reply:
Thanks, I added this to a new commit. I'm a beginner in js so didn't know about these ways but glad to see your comments which helped me improve my work)